bakery/src/cli/execute.js

```diff
@@ -125,13 +126,36 @@ const flyExecute = async (cmdArgs, { image, persist }) => {
       output: 'silent'
     })
 
-    console.log('syncing')
-    const sync = spawn('fly', [
-      'sync',
-      '-c', 'http://localhost:8080'
-    ], { stdio: 'inherit' })
-    children.push(sync)
-    await completion(sync)
+    try {
+      console.log('syncing')
+      const sync = spawn('fly', [
+        'sync',
+        '-c', 'http://localhost:8080'
+      ], { stdio: 'inherit' })
+      children.push(sync)
+      await completion(sync)
+    } catch (err) {
+      console.log('current fly cli incompatible with this concourse version')
+      console.log('syncing fly cli via direct download')
+      const printOldFlyVersion = spawn('fly', ['--version'])
```

Comment:
This works for me as far as the manual sync (e.g. when I run with fly 5.8 I end up with fly 6.1), but I didn't see the two version outputs to the screen which I think this is for?:
```
syncing
error: unknown flag `c'
current fly cli incompatible with this concourse version
syncing fly cli via direct download
logging in
logging in to team 'main'
```

Reply:
Good eye. I need to add `{ stdio: inherit }`